src/app/clusters/air-quality-server/air-quality-server.cpp

```diff
@@ -93,6 +93,11 @@ Protocols::InteractionModel::Status Instance::UpdateAirQuality(AirQualityEnum aN
         break;
     }
 
+    if (to_underlying(aNewAirQuality) > 6)
```

Comment:
Is there a reason not to put this in the switch default case?

Reply:
Yeah. It's better in the switch. See [here](https://github.com/project-chip/connectedhomeip/pull/28993/commits/ca664fabb50221311c3930917c8ee3246fb88fb8).